tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
```

Comment:
There may well be tweaks we want to make here; I tried to keep this minimal to start with.